form/models.py

```diff
@@ -6,6 +6,17 @@ class Form(models.Model):
 
     created = models.DateTimeField(auto_now_add=True)
     data = JSONField()
+    # Unique constraint to achieve “exactly-once delivery” with Amazon SQS
+    sqs_message_id = models.CharField(
+        max_length=255, blank=False, null=True, unique=True
```

Comment:
Ben and I already discussed this, but, unless there's a good reason, Django [discourages](https://docs.djangoproject.com/en/1.10/ref/models/fields/#null) using `null=True` for `CharField`